testcafe/tests/drive/public-viewer-feature.js

```diff
@@ -0,0 +1,329 @@
+  )
+  await t.fixtureCtx.vr.takeScreenshotAndUpload(
+    `${FEATURE_PREFIX}/${TEST_CREATE_FOLDER}-2`
+  )
+  console.groupEnd()
+})
+
+test(`${TEST_UPLOAD_AND_SHARE}`, async t => {
+  console.group(
+    `↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_UPLOAD_AND_SHARE} (in "${FEATURE_PREFIX} - ${TEST_CREATE_FOLDER}" folder)`
+  )
+  await privateDrivePage.goToFolder(TEST_CREATE_FOLDER)
+  await t.fixtureCtx.vr.takeScreenshotAndUpload(
+    `${FEATURE_PREFIX}/${TEST_UPLOAD_AND_SHARE}-1`
+  )
+  await privateDrivePage.uploadFiles(data.filesList)
+  //add wait to avoid thumbnail error on screenshots
+  await t.wait(5000)
+  await t.fixtureCtx.vr.setMaksCoordonnates({
+    height: 935,
+    x: 916,
+    width: 140,
+    y: 248
+  })
+  await t.fixtureCtx.vr.takeScreenshotAndUpload(
+    `${FEATURE_PREFIX}/${TEST_UPLOAD_AND_SHARE}-2`,
+    true
+  )
+  await privateDrivePage.shareFolderPublicLink()
+
+  await t.fixtureCtx.vr.setMaksCoordonnates({
+    height: 918,
+    x: 916,
+    width: 140,
+    y: 520
+  })
+  await t.fixtureCtx.vr.takeScreenshotAndUpload(
+    `${FEATURE_PREFIX}/${TEST_UPLOAD_AND_SHARE}-3`,
+    true
+  )
+
+  const link = await privateDrivePage.copyBtnShareByLink.getAttribute(
+    'data-test-url'
+  )
+  if (link) {
+    data.sharingLink = link
+    console.log(`data.sharingLink : ` + data.sharingLink)
+  }
+  console.groupEnd()
+})
+
+//************************
+// Public (no authentification) - with Download
+//************************
+fixture`${FIXTURE_PUBLIC_WITH_DL}`.page`${TESTCAFE_DRIVE_URL}/`
+  .before(async ctx => {
+    await initVR(ctx, SLUG, FIXTURE_PUBLIC_WITH_DL)
+  })
+  .beforeEach(async t => {
+    console.group(
+      `\n↳ ℹ️  no Loggin (anonymous), DOWNLOAD_PATH initialization and Navigate to link`
+    )
+    await t.useRole(Role.anonymous())
+    await t.navigateTo(data.sharingLink)
+    await publicDrivePage.waitForLoading({ isFull: true })
+
+    await setDownloadPath(data.DOWNLOAD_PATH)
+    //Init count for navigation
+    t.ctx.totalFilesCount = await publicDrivePage.getContentRowCount(
+      `${FIXTURE_PUBLIC_WITH_DL} Before`
+    )
+    console.groupEnd()
+  })
+  .afterEach(async t => {
+    console.log(
+      `↳ ℹ️  ${FEATURE_PREFIX} - Checking downloaded file for ${
+        t.ctx.fileDownloaded
+      }`
+    )
+    await checkLocalFile(`${data.DOWNLOAD_PATH}/${t.ctx.fileDownloaded}`)
+    await deleteLocalFile(`${data.DOWNLOAD_PATH}/${t.ctx.fileDownloaded}`)
+  })
+  .after(async ctx => {
+    await ctx.vr.checkRunStatus()
+  })
+
+test(`${TEST_PUBLIC_VIEWER_ZIP}`, async t => {
+  console.group(`↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_PUBLIC_VIEWER_ZIP}`)
+  //take a general screen for the shared folder :
+  await t.fixtureCtx.vr.setMaksCoordonnates({
+    height: 960,
+    x: 800,
+    width: 140,
+    y: 180
+  })
+
+  await t.fixtureCtx.vr.takeScreenshotAndUpload(
+    `${FEATURE_PREFIX}/${TEST_PUBLIC_VIEWER_ZIP}-wholeFolder`,
+    true
+  )
+  await publicViewerPage.checkViewerNavigation_vr(
```

Comment:
I use Snake (upper) casing for string constant, but I see why this is weird.
I change it to 
`${FEATURE_PREFIX}/${TEST_PUBLIC_VIEWER_ZIP}-WHOLE_FOLDER` 
but it will make the CI failed (on 1st build) as it will create a screenshot with a new name! (just saying because today build are very slow! )

Reply:
Actually I was taking about `checkViewerNavigation_vr`, I don't mind the screenshot name one.